api/cloudcontroller/ccv3/service_instance_test.go

```diff
@@ -673,12 +672,61 @@ var _ = Describe("Service Instance", func() {
 				actualRequest := requester.MakeRequestArgsForCall(0)
 				Expect(actualRequest.RequestName).To(Equal(internal.GetServiceInstanceRelationshipsSharedSpacesRequest))
 				Expect(actualRequest.URIParams).To(Equal(internal.Params{"service_instance_guid": serviceInstanceGUID}))
+				Expect(actualRequest.Query).To(ConsistOf(
+					Query{
+						Key:    FieldsSpace,
+						Values: []string{"guid", "name", "relationships.organization"},
+					},
+					Query{
+						Key:    FieldsSpaceOrganization,
+						Values: []string{"guid", "name"},
+					},
+				))
 			})
 
 			When("the request succeeds", func() {
 				BeforeEach(func() {
 					requester.MakeRequestCalls(func(params RequestParams) (JobURL, Warnings, error) {
-						json.Unmarshal([]byte(`{"data":[{"guid":"some-space-guid"}, {"guid":"some-other-space-guid"}]}`), params.ResponseBody)
+						jsonry.Unmarshal([]byte(`{
```

Comment:
This is a little unconventional. Typically we see two types of API test:
1) those that fake the Cloud Controller at the HTTP layer by providing JSON responses, such as: https://github.com/cloudfoundry/cli/blob/985a55aa2ece3151c463c025dce460ef69daa557/api/cloudcontroller/ccv3/service_broker_test.go
2) those that fake the `MakeRequest()` layer, which typically provide Go structures as responses

Given this test has a JSON response, it might be clearer if it used pattern 1

Reply:
I realise that the unconventionality was introduced before this commit...